todo/models.py

```diff
@@ -0,0 +1,25 @@
+from django.db import models
+
+
+class Tag(models.Model):
+    name = models.CharField(max_length=100)
+
+    class Meta:
+        ordering = ["name"]
+
+    def __str__(self):
+        return f"{self.name}"
+
+
+class Task(models.Model):
+    content = models.CharField(blank=True, max_length=15)
+    created_at = models.DateTimeField(auto_now_add=True)
+    deadline = models.DateTimeField()
```

Comment:
`deadline` field should be optional, according to README. Actual behavior is different from the expected:
![image](https://github.com/MykolaKlishch/py-todo-list-test-task/assets/35305198/e7008478-8530-4f6e-9dd9-cfe491897ef6)